sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2930,10 +2930,12 @@ def _provision(
                                               skip_unnecessary_provisioning)
             except locks.LockTimeout:
                 if not communicated_with_user:
-                    logger.info(f'{colorama.Fore.YELLOW}'
-                                f'Launching delayed, check concurrent tasks: '
-                                f'sky api status')
-                    communicated_with_user = True
+                    rich_utils.force_update_status(
+                        ux_utils.spinner_message('Launching delayed  ' +
+                                                 colorama.Style.RESET_ALL +
+                                                 colorama.Style.DIM +
+                                                 'Check concurrent requests: ' +
+                                                 'sky api status '))
```

Comment:
```suggestion
                        ux_utils.spinner_message(''Launching - blocked by other requests[/]  ' +
                                                 colorama.Style.DIM +
                                                 'Check requests: ' +
                                                 'sky api status' + 
                                                 colorama.Style.RESET_ALL))
```

Made a small tweak. How does it look?

Reply:
Oh, got it! Your styling string works too. Did not realize that. Changing the wording only sounds good to me : )